src/index.html

```diff
@@ -11,7 +11,151 @@
+        <label class="personal_information_label">
+          How old are You?
+          <input
+          type="number"
+          name="years-old"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+        <label class="personal_information_label">
+          Full date of birth:
+          <input
+          type="date"
+          name="dateOfBirth"
+          min="1923-01-01"
+          max="2022-01-01"
+          >
+        </label>
+        <label class="personal_information_label1">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="terms"
+          min="1923-01-01"
+          max="2022-01-01"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="registration">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label class="registration1">
+          Password:
+          <input
+          type="password"
+          name="password"
+          id="password"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="facts">
+          Do you love cats?
+          <input
+          type="radio"
+          name="cat"
+          value="Yes"
+          required
+          >
+          Yes
+          <input
+          type="radio"
+          name="cat"
+          value="No"
+          required
+          >
+          No
+        </label>
+        <label class="facts">
+         What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          required
+          >
+        </label>
+        <label class="facts">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          required
+          >
+        </label>
+        <label class="facts">
+          What are your favorite brand of cars?
+          <select multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="facts1">
+          How do you rate our work?
+          <input type="range" value="0">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="info">
+          Comments:
+          <textarea
+          name="Comments"
+          cols="20"
+          rows="2"
+          >
+          </textarea>
+        </label>
+        <label class="info1">
+         Would you recommend us?
+          <select name="recomend">
+            <option value="Yes" selected>Yes</option>
+            <option value="No">Yes</option>
```

Comment:
The second option in the 'Would you recommend us?' select element should have a value of 'No' and display 'No' as well 🤔